lib/fake-xhr/index.test.js

```diff
@@ -2568,6 +2568,32 @@ describe("FakeXMLHttpRequest", function () {
                 });
             });
         }
+    });
+
+    describe("fakeXMLHttpRequestFor", function () {
+        var win;
+
+        beforeEach(function () {
+            win = new JSDOM().window;
+            win.DOMParser = DOMParser;
 
+            var scopedSinonFakeXhr = sinonFakeXhr.fakeXMLHttpRequestFor(win);
+
+            globalXMLHttpRequest = win.XMLHttpRequest;
+            fakeXhr = scopedSinonFakeXhr.useFakeXMLHttpRequest();
+            FakeXMLHttpRequest = scopedSinonFakeXhr.FakeXMLHttpRequest;
+        });
+
+        afterEach(fakeXhrTearDown);
+
+        it("replaces global XMLHttpRequest", function () {
```

Comment:
Without reading the `beforeEach`, I think the wording is off. Isn't it more correct to actually say "replaces the XMLHttpRequest on the given Window object"? It doesn't actually replace any globals in the Node environment, which a casual reader might think when not thinking (or being tired, whatevs).

Reply:
Yes, that would make the test a lot easier to understand. I've update the PR to reflect the change.